### PR TITLE
Reword informational text in authoring commands

### PR DESCRIPTION
**Pull request by praveenrewar**
#### What this PR does / why we need it:
Reword informational text in authoring commands

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

##### Review Checklist:

- [ ] Follows the [developer guidelines](https://carvel.dev/shared/docs/latest/development_guidelines/)
- [ ] Relevant tests are added or updated
- [ ] Relevant docs in this repo added or updated
- [ ] Relevant carvel.dev docs added or updated in a separate PR and there's
  a link to that PR
- [ ] Code is at least as readable and maintainable as it was before this
  change

#### Additional documentation e.g., Proposal, usage docs, etc.:

```docs

```

### CODE REVIEW

File: cli/pkg/kctrl/cmd/app/init.go [hunk 1]
```diff
@@ -149,7 +151,9 @@ func (o *InitOptions) writeAppFile(appBuild *buildconfigs.AppBuild) error {
 	o.ui.PrintInformationalText("Successfully updated app-build.yml\n")
 	o.ui.PrintInformationalText("Successfully updated app.yml\n")
 	o.ui.PrintHeaderText("\n**Next steps**")
-	o.ui.PrintInformationalText("Created files can be consumed in following ways:\n1. Optionally, use 'kctrl dev deploy' to iterate on the app and deploy locally.\n2. Use 'kctrl app release' to release the app.\n")
+	o.ui.PrintInformationalText("Created files can be consumed in following ways:\n" +
+		"1. Optionally, use `dev deploy` command to iterate on the app and deploy locally.\n" +
+		"2. Use `app release` command to release the app.\n")
```
Comment (100mik) [inline cli/pkg/kctrl/cmd/app/init.go]: Let's remove this since the command is not mounted?

**Review by 100mik — APPROVED** [review]
Looks good but we could take care of this point as well
(Nice catch on the other error!)

File: cli/pkg/kctrl/cmd/app/init.go (hunk 1 above)
Comment (praveenrewar) [reply cli/pkg/kctrl/cmd/app/init.go]: Done

**Comment by 100mik** [comment]
Merge away once it is green!

**Pull request merged**